### ISSUE TITLE
Add DelayedDeviceMessage message

### ISSUE THREAD

**Issue by qdot** [buttplugio/buttplug#83]
@qdot commented on [Mon Jul 31 2017](https://github.com/buttplugio/buttplug-schema/issues/3)

Add a message that wraps another device message and adds a delay field (in milliseconds), allowing the wrapped message to be executed at a later time.

**Comment by blackspherefollower** [comment]
As I recall, the original use case was to make it easier to implement "vibrate for N seconds".

I think we also talked about a Repeat message, which would basically allow patterns to sent in a single message when used in conjunction with this Delay command. Would it be cleaner though to add a repeat flag to this Delay message rather than have 2 messages?

**Comment by qdot** [comment]
Yup, that's something I was considering while writing this last night, since as you said, I can't really see why you'd want to use one without the other. It'll require renaming this since it'll be more than Delayed, but I'll figure out something. Got some time anyways, this is the milestone after initial input messages ship.
